dcat/index.html

```diff
@@ -1330,7 +1329,6 @@ <h3>Class: Catalogued resource</h3>
 <a href="#Property:resource_conformsto">conformsTo</a>,
 <a href="#Property:resource_contactPoint">contact point</a>,
 <a href="#Property:resource_description">description</a>,
-<a href="#Property:resource_distribution">distribution</a>,
```

Comment:
@dr-shorthair just double-checking, the dat:distribution property is removed from here because it only applies to datasets and not catalogued resources, correct?

Reply:
Correct. It was one of the cut-and-paste debris.

Reply:
ok, thanks